### PR TITLE
reatibot-report-vc-bot-messages-to-separate-channel

### PR DESCRIPTION
**Pull request by mattmattvanvoorst**
Rework logger into a Map, so we can identify loggers better.

Our logger now logs into a specific channel (and the default logger). This allows us to more granularly log into specific channels.

VC specific message are now logged into a separate channel, to declutter the bot-log

### CODE REVIEW

**Comment by vcarl** [comment]
> 5.083 [warn] Code style issues found in the above file(s). Forgot to run Prettier?
:doge:

File: package-lock.json
Comment (vcarl) [inline package-lock.json:1]: Intentional changes? I ask cuz I don't see `package.json` changes which I'd expect

File: src/constants/channels.ts [hunk 1]
```diff
@@ -37,6 +38,7 @@ const PRODUCTION_CHANNELS = {
   techReadsAndNews: "105816607976095744",
   modLog: "591326408396111907",
   botLog: "701462381703856158",
+  vcLog: '' // TODO: add me once we have channel in prod
```
Comment (vcarl) [inline src/constants/channels.ts:41]:
```suggestion
  vcLog: "1237161125473161347"
```

File: package-lock.json
Comment (mattmattvanvoorst) [reply package-lock.json:1]: Not sure - I’m running node 18 or 20, this might be a sideffect. I’ll check why this was updated

**Review by vcarl — COMMENTED** [review]
Looks good to me but yea the formatting enforcement is complaining
```
#11 3.977 [warn] src/constants/channels.ts
#11 4.555 [warn] src/features/log.ts
#11 4.663 [warn] src/features/voice-activity.ts
#11 4.898 [warn] src/index.ts
#11 4.935 [warn] Code style issues found in the above file(s). Forgot to run Prettier?
```

File: package-lock.json
Comment (mattmattvanvoorst) [reply package-lock.json:1]: Seems like a Node 20 thing, we can ignore this

**Comment by mattmattvanvoorst** [comment]
I cant make heads or tails of that error in the Docker step. I need an adult

**Comment by vcarl** [comment]
Yup I know that one. I never got it set up right to correctly work for forks, so that error is expected. I should make an issue for it, I thought there already was one

**Review by vcarl — APPROVED** [review]
Merge at your leisure! lgtm

**Pull request merged**